src/cmd/prepare.go

```diff
@@ -37,7 +37,7 @@ var prepareTransformGitLinks = &cobra.Command{
 
 		// Perform git url transformation via regex
 		text := string(content)
-		processedText := git.MutateGitUrlsInText(host, text)
+		processedText := git.MutateGitUrlsInText(host, text, config.InitOptions.GitServer.PushUsername)
```

Comment:
This one is a little funny to me as the username is now dynamic and a `package deploy` configuration, not `package create` configuration.  Should this default to the internal user, but have an option to specify a custom one?  

Also curious how this works with external git servers--we can predict the path based off the username, is that always true in other git servers or is that something we need to talk about too?

Reply:
... and scrolling down I see the flag for the pus user option, so disregard the first point.

Reply:
It seems to be fairly consistent with all of the git servers I have used before. The path is the domain followed by either the username or org name. 

I reworded the flag message here to make it a little more obvious what to do if the repo is originally pushed under an org.

Reply:
That is how usernames work in GitLab, GitHub, BitBucket, and GitTea (and any other host based on those like CodeBerg) so it is likely good enough (and even if they do something like `https://example.com/scm/username/project.git` instead of a subdomain you should be able to use the full `https://example.com/scm/` as the `host` variable).